src/database/migrations/20220321123904-create-trip.js

```diff
@@ -0,0 +1,56 @@
+/* eslint-disable no-unused-vars */
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.createTable('trips', {
```

Comment:
The migration should also know about your tables' relationship (like you did in the model)